### DISCUSSION TITLE
Support for Treadmill Incline on other Platforms?

### DISCUSSION THREAD

**Discussion by Domoel** [cagnulein/qdomyos-zwift#2674] [General]
Hi,

is it planned to add the treadmill auto incline feature to other apps like mywhoosh? What would be necessary to support the development of this feature? Thanks!

BR
Dom

**Comment by cagnulein** [comment]
Yes I can do: let's start from a question : does the mywhoosh companion
shows realtime inclination?

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno ven 18 ott 2024 alle 18:46 Dome ***@***.***> ha
scritto:

**Reply by Domoel** [reply]
I quickly checked this and here are all available information from the companion app. 
![Screenshot_20241018_223821_MyWhoosh Link](https://github.com/user-attachments/assets/07ff6a50-02ec-4d2f-9042-df53747e0bc8)
 As you can see only elevation gained but no incline is displayed.

Can I support more towards that question / feature?

BR
Dom

**Reply by cagnulein** [reply]
So we need to analyze the protobuffer of the mywhoosh app. This will require a long hacking 

Do you know someone inside the mywhoosh team to ask some insights about this?

**Reply by Domoel** [reply]
Unfortunately not but I will try my best to find someone with these information.

**Comment by cagnulein** [comment]
That would be great, thank you!


Il giorno sab 19 ott 2024 alle 10:05 Dome ***@***.***> ha
scritto:
